src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -1516,7 +1516,7 @@ public void CancelledBuildWithUnexecutedSubmission()
         /// <summary>
         /// A canceled build
         /// </summary>
-        [Fact(Timeout = 20_000)]
+        [Fact]
```

Comment:
Could we migrate to the new xunit in a separate PR so everything's not quite so jumbled together?